src/app/datapoints-graph/model/datapoints-graph-widget.model.ts

```diff
@@ -39,6 +49,22 @@ export type DatapointsGraphWidgetConfig = {
   xAxisSplitLines?: boolean | null;
 };
 
+export type AlarmDetailsExtended = AlarmDetails & {
+  __hidden?: boolean;
+  __severity?: SeverityType[];
+  __status?: AlarmStatusType[];
+};
+
+export type EventDetailsExtended = EventDetails & {
+  __hidden?: boolean;
+};
+
+/**
+ * @description: Extended AlarmOrEvent type which includes properties from the incoming alarms/events
+ * such as severity and status.
```

Comment:
it would be better to describe what is the purpose to provide this kind of info. currently it's just rephrase of interface content.